### PR TITLE
DOCS-12782 prereq clarification

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-12782 prereq warning

### CODE REVIEW

**Review by gkop — DISMISSED** [review]

**Review by Rachelbot007 — DISMISSED** [review]
Looks good. For guided set up, is there anyway we can call out the pre-installation steps more? Maybe add a header for it, or make it bold? I see that it's already linked in the current guided set up docs, but for some reason people still miss it.

**Comment by wwarner-salesforce** [comment]
Good idea! I added headers to call out the prerequisite steps more.

**Review by Rachelbot007 — DISMISSED** [review]
Looks amazing!!!!

**Review by gkop — DISMISSED** [review]

File: modules/ROOT/pages/guided-setup.adoc [hunk 1]
```diff
@@ -4,9 +4,11 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-From the Anypoint API Community Manager control panel, open the Salesforce App Launcher and search for *API Community Manager Guided Setup*.
+== Prerequisites
+
+To set up Anypoint API Community Manager with guided setup, complete the xref:installation-prerequisites.adoc[prerequisite steps] and then follow these steps. The prerequisite steps include configuring an identity provider. Using OpenID Connect as your identity provider requires using the xref:manual-setup.adoc[manual setup] and not guided setup.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
To set up Anypoint API Community Manager with guided setup, complete the xref:installation-prerequisites.adoc[prerequisite steps] and then follow these steps. The prerequisite steps include configuring an identity provider. ```

Comment (ferferrarazzo) [reply modules/ROOT/pages/guided-setup.adoc]: Now we support OpenId in guided setup

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ Set permissions and a role for your Salesforce API Community Manager organizatio
 +
 This can be any role from the list, but cannot be none.
 +
-If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization.
+If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization by entering the name "Salesforce CRM Content User" as a string.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/installation-prerequisites.adoc]: This is not ok, a Salesforce CRM Content User is a different thing than the role, could we review this?

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/guided-setup.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Thank you! Updated to remove that.

File: modules/ROOT/pages/installation-prerequisites.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/installation-prerequisites.adoc]: Thank you! Updated per our discussion. Would you please review?

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 3]
```diff
@@ -119,7 +121,7 @@ Configure external identity in your Anypoint organization:
 . Navigate to *Access Management* > *Identity Providers* > *SAML 2.0*.
 . Set *Sign On URL* to *IdP-Initiated Login URL*.
 . Set *Sign Off URL* to *Single Logout Endpoint*.
-. Set *Issuer* to match the Salesforce account: `salesforce_org_domainname`.
+. Set *Issuer* to *salesforce_org_domainname* (such as `+https://[YourOrgDomain].my.salesforce.com+`).
 . Set the *Public Key* to the public key extracted from the `<ds:X509Certificate>` tag in the metadata XML you downloaded.
 . Set *Audience* to match the *Entity Id* you set in the Salesforce account.
 . Select *Enable new non-SSO users*.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
```

Comment (wwarner-salesforce) [reply modules/ROOT/pages/installation-prerequisites.adoc]: Thank you! Removed.

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**